### DISCUSSION TITLE
Why does special:pages does not show subfolder content?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#242] [Ask a question] [answered]
In datenstrom/community#239 I gave the hint of creating subfolder for achieving a structured url like ´website.com/folder/article`.

Because of I was not shure if this will work as described, I tried. *This* works fine. 

An unexpected challenge:

Using the `special:pages` command in an area with subfolders, only the current folder are listed. 

Same to `special:changes`.

This leads to some questions:

- Is there a special notation of commands required, to get the whole folder content including subfolders listed?
- Is there a notation to use this commands directly in a page, to create "content pages" by something like `[special:pages /folder/subfolder/] on a page as content overview page (this may make the previous question redundant)?
- Is there an option, to change the heading? The current solution displays `folder: all pages` for `/folder/subfolder/special:pages` while this should title `subfolder: all pages`

**Comment by markseuffert** [comment]
Currently the answer is no. The location argument `special:pages` is something that the [wiki plugin](https://github.com/annaesvensson/yellow-wiki) offers. The wiki plugin supports only files in a single folder. If people would like to use subfolders, then plugins have to work across multiple folders and this means some redesign in Yellow's content handling. 

Hope this answers your questions.

**Comment by BsNoSi** [comment]
Thanks, helps, hopefully others, too :)

**Comment by markseuffert** [comment, marked as answer]
In the latest Yellow there's support for wiki with subfolders. You can enable it by configuring a wiki location, that's where your wiki start page is. For example open file `system/extensions/yellow-system.ini` and change `WikiStartLocation: /wiki/`.  Now `/wiki/special:pages/` will show wiki pages in any folder.
